jax/_src/test_util.py

```diff
@@ -1136,6 +1136,10 @@ def supported(self, dtypes):
     supported = supported_dtypes()
     return type(dtypes)(d for d in dtypes if d in supported)
 
+  @_cached_property
+  def custom_floats(self):
+    return _dtypes._custom_float_dtypes
```

Comment:
Nit: `_` use. Up to you whether you want to fix or not.

Reply:
Done – listing these explicitly is slightly worse here because it means if we add another float8 flavor we'll have to remember to add it to the list here to get coverage, but that's probably OK.